### PR TITLE
Fix command message

### PR DESCRIPTION
**Pull request by TheVeryStarlk**

### CODE REVIEW

**Review by Craftplacer — APPROVED** [review]

File: Obsidian/Server.cs [hunk 1]
```diff
@@ -561,6 +561,11 @@ await player.SendSoundAsync(SoundEffectBuilder.Create(SoundId.EntitySheepAmbient
             // Just stop looping.
         }
 
+        foreach (var client in _clients)
+        {
+            client.SendPacket(new DisconnectPacket(ChatMessage.Simple("Server closed"), client.State));
+        }
+
```
Comment (Craftplacer) [inline Obsidian/Server.cs:568]: This doesn't belong in the PR

**Review by Craftplacer — CHANGES_REQUESTED** [review]

File: Obsidian/Server.cs (hunk 1 above)
Comment (TheVeryStarlk) [reply Obsidian/Server.cs:568]: Yeah sorry. I suck at Git

Comment (TheVeryStarlk) [reply Obsidian/Server.cs:568]: Will re-open the PR
